src/main.cc

```diff
@@ -1,3 +1,28 @@
+#include <algorithm>
+#include <cctype>
+#include <iomanip>
 #include <iostream>
+#include <sstream>
+#include <string>
 
-int main(int argc, char** argv) { std::cout << "Pipe network\n"; }
+#include "hydralic_sim.h"
+
+bool to_bool(std::string str) {
+  std::transform(str.begin(), str.end(), str.begin(), ::tolower);
+  std::istringstream is(str);
+  bool b = (str == "true") || (str == "t");
+  return b;
+}
+
+int main(int argc, char** argv) {
```

Comment:
You could use something like `tclap` to handle input arguments. http://tclap.sourceforge.net/